mockgen/parse.go

```diff
@@ -232,6 +232,9 @@ func (p *fileParser) parseFile(importPath string, file *ast.File) (*model.Packag
 			continue
 		}
 		i, err := p.parseInterface(ni.name.String(), importPath, ni)
+		if err == errIgnore {
```

Comment:
```suggestion
		if errors.Is(err, errConstraintInterface) {
```

This is a more safe check that will not break if, for some reason in a future update, the error gets wrapped.